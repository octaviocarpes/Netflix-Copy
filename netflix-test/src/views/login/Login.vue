<template>
  <div class="login-view">
    <div class="black-filter"></div>
    <div class="header-logo">
      <h1>NIETFLIX</h1>
    </div>
    <LoginForm />
    <footer>
      <div class="footer-content">
        <p>Created by Octavio Carpes</p>
        <ul>
          <li><a href="https://github.com/octaviocarpes">Github</a></li>
          <li>
            <a
              href="https://www.linkedin.com/in/octavio-pitrez-carpes-bbbab0148/"
              >Linkedin</a
            >
          </li>
        </ul>
        <p>
          *This is a copy made for study purposes, all rights reserved to
          Netflix
        </p>
      </div>
    </footer>
  </div>
</template>

<script>
import LoginForm from "@/components/login-form/LoginForm";

export default {
  name: "Login",
  components: {
    LoginForm
  }
};
</script>

<style lang="scss" scoped>
@import "./style.scss";
</style>
