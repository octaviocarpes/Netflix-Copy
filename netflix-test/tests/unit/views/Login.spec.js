import Vue from "vue";
import Login from "@/views/login/Login.vue";

describe("Login.spec.js", () => {
  let cmp, vm;

  beforeEach(() => {
    cmp = Vue.extend(Login);
    vm = new cmp({}).$mount();
  });

  it("has the expected html structure", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});