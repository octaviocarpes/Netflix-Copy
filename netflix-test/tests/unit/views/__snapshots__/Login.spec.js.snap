// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login.spec.js has the expected html structure 1`] = `
<div
  class="login-view"
>
  <div
    class="black-filter"
  />
   
  <div
    class="header-logo"
  >
    <h1>
      NIETFLIX
    </h1>
  </div>
   
  <div
    class="login-form-container"
  >
    <h1>
      Sign In
    </h1>
     
    <form
      class="login-inputs"
    >
      <input
        id="login-form-input"
        placeholder="Login"
        tabindex="1"
        type="text"
      />
       
      <input
        id="login-form-password-input"
        placeholder="Password"
        tabindex="2"
        type="password"
      />
       
      <button
        tabindex="3"
        type="submit"
      >
        
      Sign In
    
      </button>
    </form>
     
    <!---->
     
    <div
      class="remember-me"
    >
      <input
        id="remember-me"
        name="remember-me"
        type="checkbox"
      />
       
      <p>
        Remember me
      </p>
    </div>
     
    <div
      class="register"
    >
      
    New to Nietflix? 
      <b>
        Sign up now!
      </b>
    </div>
     
    <!---->
  </div>
   
  <footer>
    <div
      class="footer-content"
    >
      <p>
        Created by Octavio Carpes
      </p>
       
      <ul>
        <li>
          <a
            href="https://github.com/octaviocarpes"
          >
            Github
          </a>
        </li>
         
        <li>
          <a
            href="https://www.linkedin.com/in/octavio-pitrez-carpes-bbbab0148/"
          >
            Linkedin
          </a>
        </li>
      </ul>
       
      <p>
        
        *This is a copy made for study purposes, all rights reserved to
        Netflix
      
      </p>
    </div>
  </footer>
</div>
`;
